src/main/java/seedu/realodex/logic/parser/DeleteCommandParser.java

```diff
@@ -12,21 +12,45 @@
  */
 public class DeleteCommandParser implements Parser<DeleteCommand> {
 
+    public static final String INDEX_AND_NAME_PROVIDED = "Please provide either an index or a name, not both.";
+    public static final String NO_FIELDS_PROVIDED = "Please provide either an index or a name.";
```

Comment:
Move these two to `DeleteCommand`

```suggestion
    public static final String MESSAGE_INDEX_AND_NAME_PROVIDED = "Please provide either an index or a name, not both.";
    public static final String MESSAGE_NO_FIELDS_PROVIDED = "Please provide either an index or a name.";
```